doto-frontend/src/routes/Route.js

```diff
@@ -5,11 +5,38 @@ import Login from "../components/pages/Login/Login";
 import Calendar from "../components/pages/Calendar/Calendar";
 import NotFound from "../components/pages/NotFound";
 import { ThemeContext } from "../context/ThemeContext";
+import CookieManager from "../helpers/CookieManager";
 import "../tailwind-generated.css";
 
+/**
+ * When the user is redirected after they are logged in to their google account
+ * We take the current url and extract the Email and JWT Token from the query parameters
+ *
+ * There was also a weird issue where the JWT token with emails ending with "gmail.com" had a
+ * "#" character at the end but the emails ending with "aucklanduni.ac.nz" did not have this issue
+ * and that is why we are doing a hacky check to see if the email domain starts with "a".
```

Comment:
This could cause errors when you have custom email domain names under company names starting with A.

Reply:
See above